src/main/java/DateTime.java

```diff
@@ -0,0 +1,146 @@
+/**
+ * Represents a date and time object to store and convert
+ * date and time to a pleasant format e.g., 12/11/2019 1700
+ * becomes 12th November 2019, 5:00pm.
+ */
+public class DateTime {
+    private String date;
+    private String time;
+
+    public DateTime(String date, String time) {
+        this.date = date;
+        this.time = time;
+    }
+
+    public DateTime(String dateTime) throws InvalidInputException {
+        String[] splitDateAndTime = dateTime.split(" ");
+        String[] splitDate = splitDateAndTime[0].split("/");
+        String day = splitDate[0];
+        String month = splitDate[1];
+        String year = splitDate[2];
+
+        if (day.equals("1") || day.equals("21") || day.equals("31")) {
```

Comment:
Great way to add the suffix of the date since there is no java api to support this.